draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -999,7 +1028,7 @@ Type     Description                   Used In
         point in the future.  Until quantum computers become available there
         is no point in attacking the authenticity of a connection because
         there are no possibilities for exploitation.  These only occur at
-        the time of the connection, for example by mounting a man-in-the-middle (MitM)
+        the time of the connection, for example by mounting an on-path (i.e. man-in-the-middle)
```

Comment:
I suspect we'll be asked to remove "man-in-the-middle" as non-inclusive language.